modules/ROOT/pages/mutations/create.adoc

```diff
@@ -111,7 +112,7 @@ Read about xref:mutations/update.adoc#_connectorcreate_relationships[`update`] f
 
 == `connectOrCreate` relationships
 
-If a related node has the `@unique` directive defined, `connectOrCreate` can be used in a nested `create` to perform a `MERGE` operation on the related node.
+If a related node has the `@unique` directive defined, you can use `connectOrCreate` nested in a `create` to perform an operation similar to a xref:https://neo4j.com/docs/cypher-manual/current/clauses/merge/[Cypher `MERGE`] operation on the related node.
```

Comment:
Maybe worth specifying what this `create` is? I don't know if it's a mutation, but just to give an example and also fix the xref to link

```suggestion
If a related node has the `@unique` directive defined, you can use `connectOrCreate` nested in a `CREATE` mutation to perform an operation similar to a link:https://neo4j.com/docs/cypher-manual/current/clauses/merge/[Cypher `MERGE`] operation on the related node.
```